sass/responsive/_tablet.scss

```diff
@@ -147,6 +147,10 @@
             #channel-header {
                 width: calc(100% - 400px);
             }
+            //adjust file drop zone
+            #file-upload-overlay {
+                width: calc(100% - 400px);
```

Comment:
What would happen if you expand the RHS view? That increases the RHS width to 960px
![Screen Shot 2022-10-26 at 9 11 04 AM](https://user-images.githubusercontent.com/14115924/198034637-42b048bd-b36c-4703-9c49-03fd76b3571d.png)

Reply:
@BenCookie95 I just followed the channel header. I don't knew if I need to consider the RHS view expansion.

Reply:
@BenCookie95 I did a test , and the drag and drop option is not available whenever the RHS is expanded.

Reply:
I was able to test on the cloud test server. I couldn't see any issues, nice work